requirements.txt

```diff
@@ -4,4 +4,7 @@ numpy~=1.24.4
 python-dotenv==1.0.0
 bitarray==2.8.2
 pydantic==2.4.2
-fastapi~=0.104.0
\ No newline at end of file
+fastapi~=0.104.0
```

Comment:
@bmagz these go in dev.requirements.txt

We should pollute the package-level dependencies with example requirements.